skimage/exposure/_adapthist.py

```diff
@@ -25,8 +25,8 @@
 
 
 @adapt_rgb(hsv_value)
-def equalize_adapthist(image, ntiles_x=None, ntiles_y=None, clip_limit=0.01,
-                       nbins=256, kernel_size=None):
+def equalize_adapthist(image, kernel_size=None,
```

Comment:
@JDWarner should kernel_size have a default value? I'm thinking of using the same default as the original default:

`if kernel_size == None:`
 .....`kernel_size = (np.round(image.shape[0] / 8), np.round(image.shape[1] / 8))`